feature/search/src/main/java/com/terning/feature/search/search/component/SearchInternList.kt

```diff
@@ -33,12 +31,15 @@ fun SearchInternList(
                 }
             ),
             style = TerningTheme.typography.body3,
-            color = Grey400
+            color = Grey400,
+            modifier = Modifier.padding(horizontal = 24.dp)
         )
 
         LazyRow(
-            modifier = Modifier.padding(vertical = 12.dp),
+            modifier = Modifier
+                .padding(vertical = 12.dp),
             horizontalArrangement = Arrangement.spacedBy(12.dp),
+            contentPadding = androidx.compose.foundation.layout.PaddingValues(horizontal = 24.dp)
```

Comment:
`androidx.compose.foundation.layout`는 임포트로 넣어주는 게 좋을 것 같아요!
```suggestion
            modifier = Modifier
                .padding(vertical = 12.dp),
            horizontalArrangement = Arrangement.spacedBy(12.dp),
            contentPadding = PaddingValues(horizontal = 24.dp)
```